synthesis/build_defs.bzl

```diff
@@ -137,6 +137,11 @@ def _synthesize_design_impl(ctx):
         for dont_use_pattern in or_config.do_not_use_cell_list:
             dont_use_args += " -dont_use {} ".format(dont_use_pattern)
 
+    if ctx.attr.verilog_defines:
+        for define in ctx.attr.verilog_defines:
+            args.add(define)
+
+    standard_cell_black_box = ctx.actions.declare_file("{}_stdcells_blackbox.v".format(ctx.attr.name))
```

Comment:
Also how does this work if the action both generates and consumes the file? Do we need to list it?

Reply:
Also another miscopy. I added it to the output list of the action.